src/ioctls/vcpu.rs

```diff
@@ -1150,7 +1173,22 @@ impl VcpuFd {
                 KVM_EXIT_WATCHDOG => Ok(VcpuExit::Watchdog),
                 KVM_EXIT_S390_TSCH => Ok(VcpuExit::S390Tsch),
                 KVM_EXIT_EPR => Ok(VcpuExit::Epr),
-                KVM_EXIT_SYSTEM_EVENT => Ok(VcpuExit::SystemEvent),
+                KVM_EXIT_SYSTEM_EVENT => unsafe {
```

Comment:
Is there any way to test this exit code in [test_run_code](https://github.com/rust-vmm/kvm-ioctls/blob/18bc6b95a86df9feaa44d8a2ad3383a1f0d179eb/src/ioctls/vcpu.rs#L1433) on aarch64/x86?

Asking because I am curious how would that be used and also because our CI is complaining about coverage :D

Reply:
Ok，I will try to add test unit of this exit code.